src/man/sssd.conf.5.xml

```diff
@@ -2928,7 +2928,8 @@ pam_gssapi_indicators_map = sudo:pkinit, sudo-i:pkinit
                     <listitem>
                         <para>
                             Determines if user credentials are also cached
-                            in the local LDB cache.
+                            in the local LDB cache. The user credentials refer
+                            to passwords, not other authentication mechanisms.
```

Comment:
Hi,

I'd like to suggest to add `... refer to passwords, which includes the first (long term) factor of two-factor authentication, not other ....`. And `Passkey and Smartcard authentications are expected to work offline as long as a successful online authentication is recorded in the cache without additional configuration`.

bye,
Sumit

Reply:
Updated following your comments